scripts/NextJs/configs/next.config.js

```diff
@@ -21,7 +21,8 @@ const COMMIT_SHA =
 const SentryWebpackPlugin = require('@sentry/webpack-plugin');
 const fs = require('fs');
 
-// We require this to fake that our plugin matches the next version
+// Next.js requires a plugin's version to match the next.js version, so we fake
+// it here by rewriting our plugin's package.json
```

Comment:
This will be copied and live in the root of a users' nextjs project. The meaning of "ours" is not clear there, we better be more explicit.